src/components/FilterBar/SavedSearchesList/SavedSearchesListItem.react.js

```diff
@@ -7,12 +7,24 @@ export class SavedSearchesListItem extends React.Component {
     this.context.filterBarActor.loadSavedSearch(this.props.searchId);
   }
 
+  onClickDelete() {
+    this.context.filterBarActor.deleteSavedSearch(this.props.searchId);
+  }
+
   render() {
+    var liStyles = {
+          display: "inline-flex !important",
+          width: "100%",
+          marginBottom: "5px"
+        };
     return (
-      <li>
-        <a className="dynamic-text-filter" onClick={this.onClick.bind(this)} style={ {cursor: "pointer"} }>
+      <li style={liStyles}>
+        <a className="dynamic-text-filter" onClick={this.onClick.bind(this)} style={ {cursor: "pointer", marginRight: "39px"} }>
```

Comment:
Inline styles? I'm not up with what's considered the norm in React world, but this looks odd to me.